_posts/2020-01-25-community-infrastructure-learning-group.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Community Infrastructure Learning Group"
+slug: community-infrastructure-learning-group
+text: 'Learning p2p infrastructure management'
+location: Zoom Meeting ID 963940070
+locationLink: https://zoom.us/j/963940070
+date: 2020-01-25
+startTime: '14:00'
+endTime: '15:00'
+---
+
+Looking to build on last meeting by scoping out some projects related to the short term road map. Namely the pub setup script, the accompanying docs, and blog/tutorial style walk through.
+
+These seem interconnected enough that it might be good to scope them out together. open to feedback on this during the week leading up to Saturday.
```

Comment:
```suggestion
These seem interconnected enough that it might be good to scope them out together. Open to feedback on this during the week leading up to Saturday.
```